applications/SystemIdentificationApplication/tests/auxiliary_files/mixed_element_1/model_part.mdpa

```diff
@@ -0,0 +1,699 @@
+Begin Properties 0
+
+End Properties
+
+Begin Nodes
+	1	0	0	2
+	2	0	0	1.5
+	3	0	0	1
+	4	0	0	0.5
+	5	0	0	0
+	6	0	2.5	2
+	7	0	2.5	1.49375
+	8	0	2.5	0.9875
+	9	0	2.5	0.49375
+	10	0	2.5	0
+	11	3.33333	0	2
+	12	3.33333	0	1.5
+	13	3.33333	0	1
+	14	3.33333	0	0.5
+	15	3.33333	0	0
+	16	3.33333	2.5	2
+	17	3.33333	2.5	1.49375
+	18	3.33333	2.5	0.9875
+	19	3.33333	2.5	0.49375
+	20	3.33333	2.5	0
+	21	0	5	2
+	22	0	5	1.4875
+	23	0	5	0.975
+	24	0	5	0.4875
+	25	0	5	0
+	26	3.33333	5	2
+	27	3.33333	5	1.4875
+	28	3.33333	5	0.975
+	29	3.33333	5	0.4875
+	30	3.33333	5	0
+	31	6.66667	0	2
+	32	6.66667	0	1.5
+	33	6.66667	0	1
+	34	6.66667	0	0.5
+	35	6.66667	0	0
+	36	6.66667	2.5	2
+	37	6.66667	2.5	1.49375
+	38	6.66667	2.5	0.9875
+	39	6.66667	2.5	0.49375
+	40	6.66667	2.5	0
+	41	0	7.5	2
+	42	0	7.5	1.49375
+	43	0	7.5	0.9875
+	44	0	7.5	0.49375
+	45	0	7.5	0
+	46	3.33333	7.5	2
+	47	3.33333	7.5	1.49375
+	48	3.33333	7.5	0.9875
+	49	6.66667	5	2
+	50	3.33333	7.5	0.49375
+	51	6.66667	5	1.4875
+	52	6.66667	5	0.975
+	53	3.33333	7.5	0
+	54	6.66667	5	0.4875
+	55	6.66667	5	0
+	56	10	0	2
+	57	0	10	2
+	58	0	10	1.5
+	59	10	0	1.5
+	60	6.66667	7.5	2
+	61	6.66667	7.5	1.49375
+	62	0	10	1
+	63	10	0	1
+	64	6.66667	7.5	0.9875
+	65	0	10	0.5
+	66	10	0	0.5
+	67	6.66667	7.5	0.49375
+	68	10	0	0
+	69	0	10	0
+	70	6.66667	7.5	0
+	71	10	2.5	2
+	72	10	2.5	1.49375
+	73	10	2.5	0.9875
+	74	10	2.5	0.49375
+	75	10	2.5	0
+	76	3.33333	10	2
+	77	3.33333	10	1.5
+	78	3.33333	10	1
+	79	3.33333	10	0.5
+	80	3.33333	10	0
+	81	10	5	2
+	82	10	5	1.4875
+	83	10	5	0.975
+	84	10	5	0.4875
+	85	10	5	0
+	86	6.66667	10	2
+	87	6.66667	10	1.5
+	88	6.66667	10	1
+	89	6.66667	10	0.5
+	90	6.66667	10	0
+	91	10	7.5	2
+	92	10	7.5	1.49375
+	93	10	7.5	0.9875
+	94	10	7.5	0.49375
+	95	10	7.5	0
+	96	10	10	2
+	97	10	10	1.5
+	98	10	10	1
+	99	10	10	0.5
+	100	10	10	0
+End Nodes
+
+Begin Elements	SmallDisplacementElement3D8N
```

Comment:
I'd make the effort to use the geometry-based I/O together with the corresponding modelers in new additions.